changelog/29263.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
+ui: Add support for the name constraints extension to be fully specified when creating root and intermediate CA certificates.
```

Comment:
NIT: maybe add the word PKI somewhere in the description. Those familiar with it I'm sure understand that CA certificates indicate PKI, but if I were to cntrl+f the changelog I might type `PKI` to see if something in that area had changed. Something like `ui/pki: ...` or within the description itself.

Reply:
I think if we want to start changing the changelog format `ui/:item` then that's something we should talk about! I don't think we've done that here, and a one off feels a little random